trieste/utils/objectives.py

```diff
@@ -89,6 +90,10 @@ def scaled_branin(x: TensorType) -> TensorType:
 """ The global minimum of the :func:`branin` function, with shape [1] and dtype float64. """
 
 
+BRANIN_SEARCH_SPACE = Box([0.0]*2, [1.0]*2)
+""" The search space for the :func:`branin` function. """
```

Comment:
fyi you can write `Box([0.0], [1.0]) ** 2`

Reply:
thanks for the tip!